.github/workflows/ci.yml

```diff
@@ -46,15 +46,25 @@ jobs:
       - name: Test
         run: npm test
 
-      - name: Build website
+      - name: Build website (master build)
         run: |
           nvm use
           npm run docs:dist
+        if: github.ref == 'refs/heads/master'
+
+      # As above, but with an extra environment variable.
+      - name: Build website (pull request build)
+        run: |
+          nvm use
+          npm run docs:dist
+        if: github.ref != 'refs/heads/master'
+        env:
+          BASE_PATH: /prs/${{ github.event.pull_request.number }}
```

Comment:
do we need to check that this value exists? Or is it guaranteed?

Reply:
It should be guaranteed on a PR yeah